lib/net/ssh/gateway/version.rb

```diff
@@ -0,0 +1,7 @@
+module Net
+  module Ssh
+    module Gateway
+      VERSION = "1.2.0"
```

Comment:
Also, assumed that you would bump the version before releasing the gem (if it was grouped with other changes). But, I can change it to 2.0.0.